gprofiler/main.py

```diff
@@ -77,15 +79,15 @@ def _generate_output_files(
         flamegraph_path = base_filename + ".html"
         flamegraph = (
             Path(resource_path("flamegraph/flamegraph_template.html"))
-            .read_text()
```

Comment:
Which `black` version are you using? Why is it changing everything o.O

Let's ping it? (Can pin your current mypy/flake8/... as well)

Reply:
I used PyCharm reformatting

Reply:
We use `black` and linting fails because yours doesn't conform :/ configure your pycharm to use it.